### ISSUE TITLE
negate permission hits survival mode for no reason

### ISSUE THREAD

**Issue by HavenServer** [turt2live/AntiShare#132]
say I want to allow ALL item use - but block chest use only in creative.
- AntiShare.deny.use_item.chest - AntiShare.allow.use_item

This blocks chest use in _survival_ as well as creative.

Even if I add - -Antishare.affect.survival still no luck.

I would also like to propose that this amazing plugin AntiShare is the best, period, I have tried them all. However - ALL "survival" touching should be removed and all "adventure" touching should be forced creative.

This would make everything so much cleaner and being a long longtime user I believe this is the heart of this plugin. "creative mode awareness"

tl:dr AS should never ever touch survival mode.

(worldguard does that just fine... and every real server will have worldguard)
